tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -390,5 +390,44 @@ public function test_forceSave_writesNothingIfThereAreNoChanges()
 
         @unlink($configFile);
     }
+    
+    public function testFromGlobalConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+        
+        $config = new Config($globalFile, $userFile, $commonFile);
+        $config->reload();
+        
+        $configCategory = $config->getFromGlobalConfig('Category');
+        $this->assertEquals('value1', $configCategory['key1']);
+    }
+    
+    public function testFromCommonConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+    
+        $config = new Config($globalFile, $userFile, $commonFile);
+        $config->reload();
+    
+        $configCategory = $config->getFromCommonConfig('Category');
+        $this->assertEquals('valueCommon', $configCategory['key2']);
```

Comment:
Can you test against the whole section instead of just 'key2'? Ie, `array('key2' => ..., 'key3' => ...)`. Same for the other two tests. After that, I'll merge.